run_dir/design/cronjobs.html

```diff
@@ -1,28 +1,30 @@
-{% extends "base.html" %}
+{% extends "base_b5.html" %}
 {% block stuff %}
 {% import datetime %}
 <div class="container-fluid" id="content">
 	<h1 id="page_title">Production cronjobs</h1>
 	<p>The tables below show information about all cronjobs running on production servers.</p>
 	<p><em>Deactivated (i.e commented) cronjobs will appear in red.</em></p>
 
-  <div class="panel-group" id="cronjobs_accordion">
+  <div class="accordion" id="cronjobs_accordion">
   {% for server, crontab in sorted(cronjobs.items(), key=lambda x:x[1].get('last_updated', ''), reverse=True) %}
     {% set two_month_ago=str(datetime.datetime.now() - datetime.timedelta(30*2))%}
-    <div class="panel {% if crontab.get('last_updated', '') < two_month_ago %} panel-warning {% else %} panel-default {% end %}">
-      <div class="panel-heading">
-        <h4 class="panel-title">
-          <a data-toggle="collapse" data-parent="#cronjobs_accordion" href="#cronjobs_{{ server }}">
+    <div class="accordion-item accordion-item-prettify">
+			<div class="accordion-default-header {% if crontab.get('last_updated', '') < two_month_ago %} accordion-warning {% end %}">
+        <h4 class="accordion-header">
+          <a class="text-decoration-none {% if crontab.get('last_updated', '') < two_month_ago %} text-warning{% else %} text-dark{% end %} collapsed"
```

Comment:
I think it's enough with the warning color on the header background and keep the text-dark for all.

Reply:
Done!